mypy/checkexpr.py

```diff
@@ -204,7 +204,7 @@ def visit_call_expr(self, e: CallExpr, allow_none_return: bool = False) -> Type:
                         or isinstance(typ, NameExpr) and node and node.kind == nodes.TYPE_ALIAS):
                     self.msg.type_arguments_not_allowed(e)
         self.try_infer_partial_type(e)
-        callee_type = self.accept(e.callee)
+        callee_type = self.accept(e.callee, disallow_any=False)
```

Comment:
I think he means updated in typeshed. Type checking-wise, from the caller's perspective, there's no difference between an an argument of type `Any` and and argument of type `object`.

Anyway, I don't think this should be allowed.  The primary purpose of this flag is _not_ to be something that's easy to enable -- it's to provide clear, strong guarantees about how thoroughly a module is type checked.  In my mind, that sets a high bar for any exceptions we'd consider making.